.tekton/tasks/tasks-results.yaml

```diff
@@ -55,7 +45,7 @@ spec:
           npx eslint packages/ -f json > eslint-report.json || true
 
           echo "Install sonarcube scanner..."
-          npm install -g sonarqube-scanner
+          npm install -g sonarqube-scanner@3.5
```

Comment:
Were you able to figure out what was causing the issues with v4? I believe we're using v20 for coverage.

Reply:
No, I wasn't able to do that. 
We can consider this as an immediate fix and can find out the issue with node20 later. Only then we can update the version to the latest

Reply:
Maybe we can create a ticket for the version update